src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -62,8 +62,8 @@
   "minecolonies.config.listofplantables.comment": "The items and item-tags that the Florist can plant.",
 
   "item.minecolonies.scepterlumberjack": "Forest Tool",
-  "item.minecolonies.scepterlumberjack.usedstart": "First position set. Click the same position to finish.",
-  "item.minecolonies.scepterlumberjack.usedend": "Second position set. Click the same position to finish.",
+  "item.minecolonies.scepterlumberjack.usedstart": "Position A set. Select Position B with a right-click or left-click the same position to finish.",
+  "item.minecolonies.scepterlumberjack.usedend": "Position B set. Select Position A with a left-click or right-click the same position to finish.",
   "item.minecolonies.scepterlumberjack.restrictiontoobig": "An area of %d blocks is too large. The maximum restricted area for the Forester is %d blocks.",
```

Comment:
Those messages should be reversed.
`item.minecolonies.scepterlumberjack.usedstart` is displayed on right click, and
`item.minecolonies.scepterlumberjack.usedend` is displayed on left click.